From 30a72d9ded44274ab738020958a9843da7187a44 Mon Sep 17 00:00:00 2001
From: arathinx <ayyappanx.rathinam@intel.com>
Date: Mon, 4 Mar 2019 15:15:04 +0800
Subject: [PATCH] DRVLIB_SW-1088: replace lantiq,falconmx-mps2 with
 lantiq,prx300-mps2 in ltq_mps2-drv_mps_linux.c file

---
 drivers/char/ltq_mps2/drv_mps_linux.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/char/ltq_mps2/drv_mps_linux.c b/drivers/char/ltq_mps2/drv_mps_linux.c
index c5320b99439d..321fca4225c1 100644
--- a/drivers/char/ltq_mps2/drv_mps_linux.c
+++ b/drivers/char/ltq_mps2/drv_mps_linux.c
@@ -969,7 +969,7 @@ int ltq_mps_probe(struct platform_device *pdev)
 }
 
 static const struct of_device_id ltq_mps_match[] = {
-  { .compatible = "lantiq,falconmx-mps2" },
+  { .compatible = "lantiq,prx300-mps2" },
   { .compatible = "lantiq,mps2-xrx500" },
   {},
 };
